### ISSUE TITLE
Description for element <useDates> in the tag library is not clear

### ISSUE THREAD

**Issue by SJagodzinski** [SAA-SDT/eac-cpf-schema#24]
Misleading description for the usage of `<nameEntryParallel>` and it's child element `<useDates>`.

## Creator of issue
1. Silke Jagodzinski
2. TS-EAS: EAC-CPF subgroup
3. s.jagodzinski@bundesarchiv.de or silkejagodzinski@gmail.com

## The issue relates to

- [ ] EAC-CPF schema issue
- [x] EAC-CPF Tag Library issue
- [ ] EAD schema issue
- [ ] EAD Tag Library issue
- [ ] Schema issue
- [ ] Tag Library issue
- [ ] Suggestions for all schemas
- [ ] Suggestions for all Tag Libraries
- [ ] Other

## Wanted change/feature
Maybe explain that the element `<nameEntryParallel>` is meant to encode parallel names like translations or pseudonyms, as the example shows, and not changed names like birth names or maiden names.

## Context

> Some of the descriptions for the use of the tags was not very clear. In particular `<useDates>` we initially interpreted that we could use to distinguish use of specific forms of a name at a given time, such as the date range in which a married woman was known by their previous maiden name. However found it was not possible to report the `<useDates>`  as part of the `<nameEntryParallel>` when the data differs in this field – which begs the question of why would you use this tag if it cannot be used to express the differing dates in which an entity was known by various names? This would be incredibly useful for person names. 

[K. Cox]

**Comment by karinbredenberg** [comment]
Yes, clearification in TL needed.

**Comment by SJagodzinski** [comment]
Could one of the native speaker make a text draft for this issue?

**Comment by karinbredenberg** [comment]
Been reading in the TL. Just adding the parenthesis from the usage to the summary would make it more clear: (e.g., official forms of the name in different languages and/or scripts, transliterated forms of the name).
And in the usage change in the first sentence so parallel is also saying "used at the same time".

**Comment by wisserk** [comment]
<useDates> is used to accommodate date strings that are part of the nameEntry and not descriptive information about the entity that is represented by that nameEntry. It is parallel to the application of $d in MARC. I think the tag library is vague as to the intention of this element. A "do not confuse with <existDates>" phrase may help to clarify that. 

Parallel names are equivalences so therefore the <useDates> would be identical as well.

**Comment by cannedit** [comment]
As promised during the meeting on the 2nd of November I would search for some examples for this, in particular some from Belgium; unfortunately Belgium doesn't have any EAC-CPF files in the APE to which I would easily have access and unfortunately the search functionality for their EAC-CPF files on their own website has a problem at the moment (https://search.arch.be/nl/zoeken-naar-archiefvormers). 
However, I have searched for examples in the APE from other countries and - surprise - there are some nice examples from the Bundesarchiv/Berlin (attached). I also found an example from the Nationaal Archief/The Hague (also attached), but in that example just an acronym is used as "nameEntryParallel". I hope this is useful.
![nameentryparallel_1](https://user-images.githubusercontent.com/22167327/49342255-a248c380-f659-11e8-8c4f-424085d2486c.png)
![nameentryparallel_2](https://user-images.githubusercontent.com/22167327/49342256-a248c380-f659-11e8-86c2-d54d76c225df.png)
![nameentryparallel_3](https://user-images.githubusercontent.com/22167327/49342257-a248c380-f659-11e8-82ea-615e10f2be13.png)
